src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -352,9 +386,10 @@ public void setMigrationEndTimeAndStatus(String migrationId,
 						String zipFileStatus = zipFiles(type, httpContext,
 								zipFileName, title, webLinkUrl, contentAccessUrl,
```

Comment:
As the attachments handling code for zipMailMigration is inter-twined with the Resoucre migration and hence this conflict araises